### ISSUE TITLE
question about buyer email address

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#9871]
![20240913-112102](https://github.com/user-attachments/assets/48d2bccc-35ad-4519-b14f-08f9c101ac32)
The buyer email address in /order/checkout-forms is the real email address? If we send a email into this email address, will buyer can get the email?

**Comment by Lukasz-Zurek** [comment]
Yes, real, hashed email. You can read more about this [here](https://help.allegro.com/sell/en/a/how-we-display-email-addresses-in-the-correspondence-within-allegro-K6VzjEmX8h4).

**Comment by yangcen6** [comment]
So we can send the email to buyer with the email adsress?

**Comment by Lukasz-Zurek** [comment]
Yes.

**Comment by yangcen6** [comment]
But for the same buyer, there is a different email address in different orders. So we can also sent the email to the same buyer with our system?

**Comment by Lukasz-Zurek** [comment]
@yangcen6 I have sent you a link to Allegro Help, where it is described. It shouldn't be a problem.
